### ISSUE TITLE
a11y support

### ISSUE THREAD

**Issue by Mister-Hope** [dimsemenov/PhotoSwipe#1869]
It's always good to have a better a11y support, but v5 seems just drop it. I cannot get any graceful way to set `ariaLabel` for each button, or title.

E.g.:

https://github.com/dimsemenov/PhotoSwipe/blob/afaa07e6df27edd37b03b901b00cd08e5b73b49b/src/js/ui/button-arrow.js#L21

**Comment by dimsemenov** [comment]
I wasn't able to find a screen reader that didn't read `title` attribute for a `<button>`. In what exactly screen reader are you getting the issue?

I can set `aria-label` to the same value as `title` if it helps some automated accessibility check tools though. Adding a filter that allows modifying button element is probably not a bad idea too.

**Comment by Mister-Hope** [comment]
Nope, I mean I may want to modify this, not everyone on the world use English.

**Comment by dimsemenov** [comment]
Oh, you mean localization, there is a section for that on [the options page of docs](https://photoswipe.com/options/#translating).
